cli/azd/internal/repository/app_init.go

```diff
@@ -290,6 +291,35 @@ func (i *Initializer) InitFromApp(
 		})
 	}
 
+	for _, proj := range projects {
+		isSpringBootProject := false
+		mavenProject, ok := proj.RawProject.(appdetect.MavenProject)
+		if !ok {
+			continue
+		}
+		for _, dep := range mavenProject.Build.Plugins {
+			if dep.GroupId == "org.springframework.boot" &&
+				dep.ArtifactId == "spring-boot-maven-plugin" {
+				isSpringBootProject = true
+				break
+			}
+		}
+		if !isSpringBootProject {
+			continue
+		}
+		for _, dep := range mavenProject.Dependencies {
+			name := dep.GroupId + ":" + dep.ArtifactId
+			switch name {
+			case "org.postgresql:postgresql",
+				"com.azure.spring:spring-cloud-azure-starter-jdbc-postgresql":
```

Comment:
Is it possible for both these dependencies to exist at the same time, and if so -- do we only want to generate once?